website/src/components/ProductPopover.tsx

```diff
@@ -11,7 +11,7 @@ interface ProductFeatureOrUseCase {
 export const PRODUCT_FEATURES: ProductFeatureOrUseCase[] = [
     { text: 'Code search', to: '/product/code-search-navigation' },
     { text: 'Code review', to: '/product/code-review' },
-    { text: 'Automation', to: '/product/automation' },
+    { text: 'Code change management', to: '/product/code-change-management' },
```

Comment:
There's a placement issue with the dropdown that showed up.
Before:
![image](https://user-images.githubusercontent.com/1559622/74807860-5b66de00-529e-11ea-8441-36f3abfd05ab.png)

After:
![image](https://user-images.githubusercontent.com/1559622/74807882-6457af80-529e-11ea-8a46-953da5606da0.png)

Reply:
Hmm, I am not seeing this. I'll ping you when it's deployed to see if you are still seeing this problem.